scss/civicrm/contact/_header.scss

```diff
@@ -15,13 +15,16 @@
   vertical-align: top;
 }
 
-.crm-summary-contactphoto {
-  border-radius: 50%;
+#crm-contact-thumbnail { 
   float: left;
-  height: $contact-avatar;
   margin: 10px $contact-avatar/2 0;
   padding-top: 5px;
-  width: $contact-avatar;
+
+  img {
```

Comment:
would a stricter `> img` selector work the same?

Reply:
Nope. It won't work. `img` is not an immediate child of  `.crm-summary-contactphoto`